.gitignore

```
@@ -3,4 +3,6 @@
 .packages
 build/
 # If you're building an application, you may want to check-in your pubspec.lock
-pubspec.lock
\ No newline at end of file
+pubspec.lock
+
+.idea
```

just adding a new line at the end of file
```suggestion
pubspec.lock
.idea

```